XML/TestCases/FunctionalTests-KernelDebug.xml

```diff
@@ -12,4 +12,17 @@
             <SetupType>OneVM</SetupType>
         </SetupConfig>
     </test>
+    <test>
+        <testName>VERIFY-HPC-VM</testName>
+        <testScript>VERIFY-HPC-VM.ps1</testScript>
```

Comment:
if no special handle logic in ps1 file, use bash script directly, it is good enough

Reply:
I will keep this format because I will add one or 2 bash scripts in future.